src/device-api/v2.ts

```diff
@@ -509,4 +512,28 @@ export function createV2Api(router: Router, applications: ApplicationManager) {
 			});
 		}
 	});
+
+	router.post('/v2/journal-logs', (req, res) => {
+		try {
+			const all = checkTruthy(req.body.all) || false;
+			const follow = checkTruthy(req.body.follow) || false;
+			const count = checkInt(req.body.count, { positive: true }) || undefined;
+			const unit = req.body.unit;
+
+			const journald = getJournaldStream({ all, follow, count, unit });
+			res.status(200);
+			journald.pipe(res);
+			res.on('close', () => {
+				journald.destroy();
```

Comment:
This is a `destroy()` call on the child process stream. Does the process gets killed too?

Reply:
It does in this case, but thinking about it, relying on that shouldn't really be done. Will change the journald module to also return the spawn handle, so it can be closed explicitly.